cachi2/core/package_managers/pip.py

```diff
@@ -1289,7 +1289,7 @@ def from_line(cls, line: str, options: list[str]) -> "PipRequirement":
         :return: PipRequirement instance
         """
         to_be_parsed = line
-        qualifiers: dict[str, str]  = {}
+        qualifiers: dict[str, str] = {}
```

Comment:
This one line fix belongs to the previous patch which introduced the E221 styling issue.